nuxeo-core/nuxeo-core-management-test/src/test/java/org/nuxeo/ecm/core/management/test/probes/TestProbes.java

```diff
@@ -79,4 +94,68 @@ public void testService() {
 
     }
 
+    @Test
+    public void testHealthCheck() throws IOException {
+
+        Collection<ProbeInfo> healthCheckProbes = pm.getHealthCheckProbes();
+        assertEquals(2, healthCheckProbes.size());
+
+        HealthCheckResult result = pm.getOrRunHealthChecks();
+        assertTrue(result.isHealthy());
+        assertEquals("{\"runtimeStatus\":\"ok\",\"repositoryStatus\":\"ok\"}", result.toJson());
```

Comment:
I think it's safer to compare Field by Field on the JSON. If you change the serializer, the JSON might still be the expected one the the order might be different, causing this test to fail.

e.g.: 
```
assertEquals("ok", result.get("runtimeStatus"));
```